src/routes/committee/schedule/availability/[rehearsalId]/+page.svelte

```diff
@@ -0,0 +1,72 @@
+<script lang="ts">
+	import type { newAvailability } from '$lib/types/domain/availability'
+	import type { rehearsal } from '$lib/types/domain/rehearsal'
+	import type { user } from '$lib/types/domain/user'
+	import { getTimeString } from '$lib/util/timeString'
+	import {
+		Button,
+		Grid,
+		Modal,
+		StructuredList,
+		StructuredListCell,
+		StructuredListHead,
+		StructuredListRow
+	} from 'carbon-components-svelte'
+	import { Chat } from 'carbon-icons-svelte'
+
+	export let data: {
+		users: { [id: string]: user }
+		availabilities: { [id: string]: newAvailability }
```

Comment:
Hier zou ik dan juist wel gewoon de goede type gebruiken, en dan in page.ts de juiste data inladen natuurlijk (want daar heb je vgm wel toegang tot het id)